src/Intents/INPerson.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using CoreGraphics;
+using Foundation;
+using ObjCRuntime;
+using System.Runtime.Versioning;
+
+#nullable enable
+
+namespace Intents
+{
+
+#if !TVOS
+	public partial class INPerson
+	{
+
+#if !NET
+	[Introduced (PlatformName.iOS, 15,0)]
```

Comment:
The indentation is still off for all the attributes in this file (looks like they should be indented one more level).